contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -507,8 +509,8 @@ resolve_numeric_typmod_from_exp(Node *expr)
 			 * Refer to details of precision and scale calculation in the following link:
 			 * https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/data-types/precision-scale-and-length-transact-sql.md
 			 */
-			has_aggregate_operand = arg1->type == T_Aggref ||
-						(list_length(op->args) == 2 && arg2->type == T_Aggref);
+			if(arg1->type == T_Aggref)
```

Comment:
space after if, also why there was a warning to start with, variable is initialised upfront

Reply:
removed that, all the changes in this commit. If some variable is getting intialised in if-else block then compiler is throwing warning that it can be possible that the variable may remain unintialised.